client/src/pages/Login.vue

```diff
@@ -80,20 +77,46 @@
 </template>
 
 <script>
+import gql from "graphql-tag";
+
 export default {
   name: "PageLogin",
   data() {
     return {
       isPwd: true,
       form: {
-        username: "",
+        email: "",
         password: ""
       },
       error: "",
       loading: false
     };
   },
   methods: {
+    async login() {
+      const loginResult = await this.$apollo
+        .mutate({
+          mutation: gql`
+            mutation($email: String!, $password: String!) {
+              login(email: $email, password: $password) {
+                id
+                name
+                username
+              }
+            }
+          `,
+          variables: {
+            email: this.form.email,
+            password: this.form.password
+          }
+        })
+        .then(data => {
+          console.log(data);
+        })
+        .catch(error => {
+          console.error(error);
+        });
+    },
     onSubmit() {
```

Comment:
It looks like onSubmit is dead code and should be deleted.

Reply:
Excellent spot. I forgot about this part.

Reply:
The old `onSubmit()` method has been removed.